lib/CL/pocl_util.c

```diff
@@ -24,12 +24,24 @@
    IN THE SOFTWARE.
 */
 
+#define _BSD_SOURCE
+#define _DEFAULT_SOURCE
+#define _POSIX_C_SOURCE 200809L
+
 #include <errno.h>
 #include <limits.h>
 #include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
 
+#if defined(__FreeBSD__)
+#include <stdlib.h>
+#elif defined(__WIN32)
```

Comment:
I think, MSVC does not define `__win32`.
```suggestion
#elif defined(_WIN32)
```